deployment/azuredeploy.json

```diff
@@ -1,523 +1,615 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "appRegistrationClientId": {
-            "type": "string",
-            "defaultValue": "dc0fcdd9-727f-47a3-8f15-4dcb813919fc"
-        },
-        "appRegistrationApiURI": {
-            "type": "string",
-            "defaultValue": "api://10ef1e6b-1171-4888-92d5-c661b428796b"
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",

```

Comment:
Why this change? Was this indentation different from the other lines in the file.

Reply:
I highly apologize but there are simply too many indentation changes in the file. It is not quite possible for me to filter and review the code changes from it. I'd request you to please revert any unnecessary changes from this file to indicate the ones of importance. We can take up the indentation changes a separate commit altogether.

Reply:
I'll fix the indentation.